src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt

```diff
@@ -0,0 +1,16 @@
+package com.openlattice.chronicle.notifications
+
+/**
+ *  @author Todd Bergman <todd@openlattice.com>
+ *  https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-
+ *  Finalized Message Delivery Status
+ *  After a message has been sent, the following final delivery status options are possible:
+ */
+
+enum class NotificationStatus {
```

Comment:
This is a duplicate of `MessageStatus`. Do you need both?

https://github.com/openlattice/chronicle-api/blob/develop/src/main/java/com/openlattice/chronicle/data/MessageStatus.kt

Reply:
MessageStatus is not in v3 branch